.prettierignore

```diff
@@ -71,6 +71,9 @@ docker-compose.override.yml
 /app/javascript/mastodon/locales
 /config/locales
 
+# Ignore vendored CSS reset
+app/javascript/styles/mastodon/reset.scss
+
```

Comment:
Shouldn't this also include glitch-soc's `app/javascript/flavours/glitch/styles/reset.scss`?

Reply:
You're right, I forgot that. I also forgot to port 0c9d455ea5540e854c2d879cd1df015ea493f622 as a whole.